google/cloud/bigtable/row_set.py

```diff
@@ -137,7 +137,7 @@ def _update_message_request(self, message):
         :param message: The ``ReadRowsRequest`` protobuf
         """
         for each in self.row_keys:
-            message.rows.row_keys.append(_to_bytes(each))
+            message.rows.row_keys._pb.append(_to_bytes(each))
```

Comment:
is it at all possible we could run into the same issues we same here with lazy eval? https://github.com/googleapis/python-bigtable/pull/359

Reply:
I wouldn't expect so. We have applied a similar patch to firestore and pubsub libraries. `row_keys.append` calls proto plus which ultimately will interact with `_pb`. The way it does this though is not fast. :(